lib/screens/StartupScreen.dart

```diff
@@ -973,4 +1241,60 @@ class _StartPageState extends State<StartPage> {
       ],
     );
   }
+
+  bool areFieldsValid() {
+    bool toReturn = true;
+    if (_firstName.value.text.isEmpty) {
+      toReturn = false;
+      unfilledFirstName = true;
+    }
+    if (_lastName.value.text.isEmpty) {
+      toReturn = false;
+      unfilledLastName = true;
+    }
+    if (_email.value.text.isEmpty) {
+      toReturn = false;
+      unfilledEmail = true;
+    }
+    if (_password.value.text.isEmpty) {
+      toReturn = false;
+      unfilledPassword = true;
+    }
+    if (!isEmail(_email.value.text)) {
+      toReturn = false;
+      errorMessage = 'Email must be in valid form';
+      unfilledEmail = true;
+    }
+    return toReturn;
+  }
+
+  void clearFields() {
+    unfilledEmail = false;
+    unfilledPassword = false;
+    unfilledFirstName = false;
+    unfilledLastName = false;
+    _email.clear();
+    _password.clear();
+    _firstName.clear();
+    _lastName.clear();
+  }
+
+  bool status(http.Response res) {
```

Comment:
You have this defined twice in the same file with the same name; pls fix. Whether you go with the approach I suggested of having this return a status or no, pls update the names to reflect that one is for register and one is for login.